src/invisible/wrapCombineReducers/index.js

```diff
@@ -0,0 +1,44 @@
+import createTypes from '../../creators/createTypes';
+
+import commonReducer, { defaultActionNames } from './commonReducer';
+
+const INVISIBLE_NAMESPACE = '$INVISIBLE';
+
+const getSuperNamespace = actionName => actionName.slice(0, actionName.indexOf(':'));
+const shouldBeExtended = action => getSuperNamespace(action.type) === INVISIBLE_NAMESPACE;
+
+const getSliceName = (action, reducerObject) => {
+  let sliceName = action.type
+    .slice(action.type.indexOf(':') + 1, action.type.indexOf('/'))
+    .replace('#', '')
+    .toLowerCase();
+  Object.keys(reducerObject).forEach(reducerName => {
+    if (reducerName.toLowerCase() === sliceName) sliceName = reducerName;
+  });
+  return sliceName;
+};
+
+const formatActionName = actionName => actionName.slice(actionName.indexOf('/') + 1);
```

Comment:
should we declare `/` and `:` as constants?

Reply:
Makes sense !